src/codegen/Common.ts

```diff
@@ -427,7 +427,37 @@ const common = {
           maxHeight: 2,
         },
       }),
-
+  ILikertColumnProperties: () =>
+    new CG.obj(
+      new CG.prop(
+        'columns',
+        new CG.arr(
+          new CG.obj(
+            new CG.prop(
+              'value',
+              new CG.union(new CG.str().setPattern(/^\d+$/), new CG.num())
+                .setTitle('Value')
+                .setDescription('The value of the answer column'),
+            ),
+            new CG.prop(
+              'addLeftDivider',
```

Comment:
Personally I would skip the word 'add' here. It doesn't add anything (pun intended) that wouldn't be just as clear if the props was named `leftDivider` and `righDivider`.

In the same vein, maybe instead of having two props for this, we could have one? `divider: "left"`, `divider: "right"` or `divider: "both"`?

Reply:
I can change it to be one property if that is preferable, either way I do agree that the "add" part is redundant.

Reply:
I'm not sure what's best either. I think even `before` and `after` might be preferred, as we technically support right-to-left languages as well. Not sure if that would change anything here, and I don't expect you to test for that, so go with what you want. :raised_hands:

Reply:
In the case of RTL `before` and `after` make more sense to than `left` and `right`. I will look into it :)